client/components/follow-button/style.scss

```diff
@@ -1,3 +1,5 @@
+$follow-button-gray-disabled: lighten( $gray, 20% );
```

Comment:
This has been merged, but we should just use the original color functions instead of creating new variables outside of `colors.scss`.